cmake/fw_zip.cmake

```diff
@@ -0,0 +1,31 @@
+
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+function(generate_dfu_zip zip_out bin_file_list deps)
```

Comment:
What will happen when this function is called twice, for example in order to create two slightly different dfu zips for uploading.

In case both depends on common target, let's say Zephyr hex, then this code will fail with duplicate custom target.

Reply:
I consider this a sane limitation. However I could do something like replace `/` in the absolute path to the file with `_` and then use this as the target name.

Reply:
I did this ^^, please check it